docs/video/smoothie/recipe.md

```diff
@@ -5,30 +5,42 @@ icon: simple/googlesearchconsole
 
 # Smoothie recipe
 
-The recipe (config file) is Smoothie's hardest part of it's learning curve, reading this and messing around with short clips is the best way to familiarize yourself with it. 
+The recipe (config file) is the hardest part of Smoothie's learning curve, reading this and messing around with short clips is the best way to familiarize yourself with it. 
 
+<!--
 The values which can be turned on or off (booleans) have a bunch of aliases for convenience, I advise typing y/n or 1/0 for short
 
-If there are some feature you're not interested in using, feel free to remove them from the file, it won't break a thing and act like you disabled it(1)
+I personally think this can go unmentioned, if you disagree and want to mention it, I'd recommend removing the term "booleans" (might be confusing for noobs)
+-->
+
+If there are features you're not interested in using, feel free to remove them from the file, it won't break a thing and act like you disabled it(1)
 { .annotate }
 
-1. Smoothie loads `defaults.ini` into a hashmap, which is just like the user's `recipe.ini` but with everything disabled / max compatibility, then it overrides the values in that hasmap that exist with `recipe.ini` / whatever specified ini file
+1. Smoothie loads `defaults.ini`, which is just like the user's `recipe.ini` but with everything disabled / max compatibility, then it overrides those value with the existing ones in `recipe.ini`. Feel free to edit `defaults.ini` so you can hide options / categories you never change in `recipe.ini`!
```

Comment:
I'm against supporting people to edit defaults.ini since that's already the backup of the recipe, you should just use recipe.ini as the intended user facing config file